eos-migrate-image-defaults.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Migrate eos-image-defaults to /var/lib
+ConditionPathIsDirectory=/var/eos-image-defaults
+ConditionPathIsSymbolicLink=!/var/eos-image-defaults
```

Comment:
You also could add `ConditionNeedsUpdate=/var` here and only run this on upgrades.